sumologic/data_source_sumologic_cse_log_mapping_vendor_product.go

```diff
@@ -0,0 +1,53 @@
+package sumologic
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceCSELogMappingVendorAndProduct() *schema.Resource {
```

Comment:
We need an acceptance test for this data source.

Reply:
https://github.com/SumoLogic/terraform-provider-sumologic/pull/284/commits/a44f995f197a0d27ca41d773aa10652ebeba4e48